src/Lean/LocalContext.lean

```diff
@@ -177,9 +177,10 @@ def mkLocalDecl (lctx : LocalContext) (fvarId : FVarId) (userName : Name) (type
     let decl := LocalDecl.cdecl idx fvarId userName type bi kind
     { fvarIdToDecl := map.insert fvarId decl, decls := decls.push decl }
 
+-- `mkLocalDecl` without `kind`
 @[export lean_local_ctx_mk_local_decl]
-private def mkLocalDeclExported (lctx : LocalContext) (fvarId : FVarId) (userName : Name) (type : Expr) (bi : BinderInfo) (kind : LocalDeclKind := .default) : LocalContext :=
-  mkLocalDecl lctx fvarId userName type bi kind
+private def mkLocalDeclExported (lctx : LocalContext) (fvarId : FVarId) (userName : Name) (type : Expr) (bi : BinderInfo) : LocalContext :=
+  mkLocalDecl lctx fvarId userName type bi
```

Comment:
@gebner I assume this was the original intention?

Reply:
Ah yes, must have been a copy & paste error.  The `mkLetDeclExported` below does the right thing.